.circleci/config.yml

```diff
@@ -253,16 +253,22 @@ workflows:
           name: test-linux-<< matrix.python-version >> | << matrix.dependency-specifiers >>
           matrix:
             parameters:
-              python-version: &python-versions ["3.9", "3.10", "3.11", "3.12", &latest-python "3.13"]
+              python-version: &python-versions ["3.10", "3.11", "3.12", "3.13", &latest-python "3.14"]
               dependency-specifiers:
                 - "dwave-cloud-client~=0.12.0"
                 - "dwave-cloud-client~=0.13.6"
-                - "dwave-cloud-client~=0.14.0"
+                - "dwave-cloud-client~=0.14.1"
               integration-test-python-version: &integration-python-versions [*latest-python]
             exclude:
               - python-version: "3.13"
                 dependency-specifiers: "dwave-cloud-client~=0.12.0"
-                integration-test-python-version: "3.13"
+                integration-test-python-version: *latest-python
+              - python-version: "3.14"
+                dependency-specifiers: "dwave-cloud-client~=0.12.0"
+                integration-test-python-version: *latest-python
+              - python-version: "3.14"
+                dependency-specifiers: "dwave-cloud-client~=0.13.6"
+                integration-test-python-version: *latest-python
```

Comment:
Not sure I understand what this even does.

Reply:
We're excluding some combinations that are not supported (very old client on a newer python).